styles.css

```diff
@@ -86,3 +88,324 @@ ul {
+  width: 90%;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 100%;
+  border-radius: 8px;
+  gap: 20px;
+  margin-bottom: 50px;
+}
+
+.grid-item {
+  margin-left: 20px;
+  height: 474px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.item {
+  width: 100%;
+  background-color: #ebf0ee;
+  height: 252px;
+  border-radius: 8px;
+}
+
+.title {
+  margin: 16px 18px;
+  width: 295px;
+  height: 57px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 28px;
+  line-height: 32px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+}
+
+.tagcontainer {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 5px;
+  gap: 20px;
+  width: 329px;
+  height: 32px;
+}
+
+.label-project {
+  background-color: #ebf0ee;
+  color: #3a4a42;
+}
+
+.see-post {
+  margin: 30px 106px 24px 103px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+}
+
+.see-button {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.see-post :link,
+.see-post:visited {
+  font-weight: 500;
+  line-height: 2.4rem;
+  letter-spacing: 0.03em;
+  text-align: center;
+  text-decoration: none;
+  color: #fff;
+  padding: 0.8rem 1.2rem;
+  background: #36b37f;
+  display: inline-block;
+  border-radius: 4px;
+  margin: 0 auto;
+}
+
+.see-button:hover {
+  color: #36b37f;
+}
+
+.see-post:link,
+.see-post:visited,
+.see-post:hover {
+  font-weight: 500;
+  line-height: 2.4rem;
+  letter-spacing: 0.03em;
+  text-align: center;
+  text-decoration: none;
+  color: #fff;
+  padding: 0.8rem 1.2rem;
+  background: #36b37f;
+  display: inline-block;
+  border-radius: 4px;
+  margin: 0 auto;
+}
+
+.see-post:active {
+  background: #008552;
+}
+
+.grid-container-another {
+  margin: 10px 10px;
+  width: 90%;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 100%;
+  border-radius: 8px;
+  gap: 20px;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 10px 20px;
+  color: #172b4d;
+}
+
+.right {
+  margin-left: 200px;
+  padding: -5px;
+  padding-left: 100px;
+}
+
+.left {
+  margin-right: 270px;
+  padding-top: -70px;
+}
+
+.about {
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.about-myself {
+  width: 351px;
+  height: 194px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+}
+
+.get-my-resume {
+  margin: 10px 106px 24px 103px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 153px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: #fff;
+}
```

Comment:
- [ ] to completely fix your button to match the figma design kindly add the below piece of code, currently your button has borders and it isn't centered and this will cemter it and remove borders :+1:
 - `border: none;`
 - `justify-content: center;`
> - figma design to be achieved
![s-btn](https://user-images.githubusercontent.com/89055502/235324435-98f3d106-3043-4870-bf08-e73657a6c767.png)